### DISCUSSION TITLE
治安局的各位直接出现在绳网上是不是不太好？

### DISCUSSION THREAD

**Discussion by Bisaqulaor** [share121/inter-knot#4729] [灌水]
如题，逛了一圈发现好多治安局的警官实名上网啊，虽然早就知道绳网上会有警察来看，但是咱们这多少也是个灰产，警官出现在这里还是略有一点慌张的
![1725937369216](https://github.com/user-attachments/assets/af23500f-0a5c-465b-97a5-e913662ead46)

**Comment by Kiana-K423-1207** [comment]
没事，查不到的（doge

**Comment by 1luik** [comment]
你应该发https://github.com/share121/inter-knot/issues而不是绳网

**Comment by HollowInvestigativeAssociation** [comment]
官方人员也有私生活，机构无权对任何人员的私生活进行干扰，此外，HIA正在招募新调查员，有意者请到光映广场HIA分部进行面试
